### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by thisandagain**

### CODE REVIEW

**Comment by thisandagain** [comment]
@gvn @k88hudson R?

File: app/src/main/res/values-en/strings.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Webmaker</string>
+    <string name="action_play">Play</string>
+    <string name="action_remix">Remix</string>
+    <string name="action_settings">Settings</string>
+    <string name="action_share">Share</string>
+    <string name="ga_appName" translatable="false"></string>
```
Comment (k88hudson) [inline app/src/main/res/values-en/strings.xml]: is this supposed to be here?

**Comment by k88hudson** [comment]
Looks good, I added a note I wasn't sure about

File: app/src/main/res/values-en/strings.xml (hunk 1 above)
Comment (thisandagain) [inline app/src/main/res/values-en/strings.xml]: Good catch. No... will remove.

**Pull request merged**